apps/researcher/src/lib/community.ts

```diff
@@ -28,8 +31,58 @@ export async function getMemberships(
   });
 }
 
-export function getAllCommunities(): Promise<Community[]> {
-  return clerkClient.organizations.getOrganizationList();
+export enum CommunitySortBy {
+  NameAsc = 'nameAsc',
+  NameDesc = 'nameDesc',
+  CreatedAtDesc = 'createdAtDesc',
+  MembershipCountDesc = 'membershipCountDesc',
+}
+
+export const defaultCommunitySortBy: CommunitySortBy =
+  CommunitySortBy.CreatedAtDesc;
+
+export function sortCommunities(
```

Comment:
Or just `sort` (since the file already makes the scope clear - it's about communities)?